src/sentry/lang/javascript/processing.py

```diff
@@ -145,7 +145,13 @@ def _handles_frame(frame):
     if not frame:
         return False
 
-    return frame.get("abs_path") is not None
+    # skip frames without an `abs_path`
+    if (abs_path := frame.get("abs_path")) is None:
+        return False
+    # skip "native" frames without a line
+    if abs_path == "native" and frame.get("lineno", 0) == 0:
```

Comment:
We should add also `"[native code]"`.

https://github.com/getsentry/sentry-react-native/blob/03e054f4876690e6b8b944d54474b371922fb18a/src/js/integrations/rewriteframes.ts#L18

Reply:
Thanks.